browser/about_flags.cc

```diff
@@ -370,6 +378,17 @@ constexpr char kFileSystemAccessAPIDescription[] =
 #define BRAVE_TRANSLATE_GO_FEATURE_ENTRIES
 #endif  // BUILDFLAG(ENABLE_BRAVE_TRANSLATE_GO)
 
+#if defined(TOOLKIT_VIEWS)
+#define BRAVE_TOOLKIT_VIEWS_FEATURE_ENTRIES                   \
```

Comment:
I think we can think separately between feature itself and its user (mute button).
Then, we can define feature w/o checking tookit view. WDYT?

Reply:
updated to remove